xformers/components/feedforward/mlp.py

```diff
@@ -9,15 +9,23 @@
 import torch
 import torch.nn as nn
 
+import xformers
 from xformers.components import Activation, build_activation
 from xformers.components.feedforward import Feedforward, FeedforwardConfig
 
+if xformers._is_functorch_available:
+    from xformers.components.nvfuser.bias_act_dropout import (
+        NVFusedBiasActivationDropout,
+    )
+
 from . import register_feedforward
 
 
 @dataclass
 class MlpConfig(FeedforwardConfig):
     hidden_layer_multiplier: int
+    bias: bool
+    nvfuser: bool
```

Comment:
Do we need this? Don't we just default to the nvfused implementation when functorch is available?

Reply:
Good catch! I forgot to remove this.